package.json

```diff
@@ -55,24 +55,24 @@
     "styled-components": "^5.1.0"
   },
   "dependencies": {
-    "grommet-icons": "^4.10.0",
+    "grommet-icons": "^4.12.0",
     "hoist-non-react-statics": "^3.2.0",
     "markdown-to-jsx": "^7.2.0",
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
-    "@babel/cli": "^7.23.4",
-    "@babel/core": "^7.23.6",
-    "@babel/eslint-parser": "^7.23.3",
-    "@babel/node": "^7.22.19",
+    "@babel/cli": "^7.23.9",
+    "@babel/core": "^7.23.9",
+    "@babel/eslint-parser": "^7.23.10",
+    "@babel/node": "^7.23.9",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-do-expressions": "^7.23.3",
     "@babel/plugin-proposal-export-default-from": "^7.23.3",
     "@babel/plugin-proposal-logical-assignment-operators": "^7.20.7",
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@babel/plugin-proposal-pipeline-operator": "^7.23.3",
-    "@babel/preset-env": "^7.23.6",
+    "@babel/preset-env": "^7.23.9",
     "@babel/preset-react": "^7.23.3",
     "@storybook/addon-a11y": "7.0.20",
```

Comment:
Is the latest storybook version not working?
I know it hasn't been working the past times we tried to upgrade but just want to confirm